drivers/wdm/audio/backpln/portcls/pin_wavert.cpp

```diff
@@ -311,18 +311,28 @@ CPortPinWaveRT::DeviceIoControl(
     }
     else if (IoStack->Parameters.DeviceIoControl.IoControlCode == IOCTL_KS_ENABLE_EVENT)
     {
-        /// FIXME
-        /// handle enable event
+        /* FIXME UNIMPLEMENTED */
+        DPRINT1("[portcls] IOCTL_KS_ENABLE_EVENT is UNIMPLEMENTED!");
```

Comment:
Need `\n`
Also the [portcls] part is unnecessary, DPRINT prints the source file/line